### ISSUE TITLE
There is no  jsoneditor-icons.png in bower_components/jsoneditor

### ISSUE THREAD

**Issue by 6174** [Glavin001/ember-jsoneditor#10]
After I do  Ember install  ember-jsoneditor

ember-cli show error

``` shell
/Users/admin/www/idcos/hf-csa-web/tmp/funnel-input_base_path-VkNgXpSw.tmp/0/bower_components/jsoneditor/dist/img/jsoneditor-icons.png'
```

I can find a svg file in the directory

**Comment by sbsurf** [comment]
+1

**Comment by Glavin001** [comment]
Fixed by #9.
